windows/src/desktop/kmshell/xml/config.css

```diff
@@ -1235,7 +1205,7 @@ th
   padding-top: 10px;
   position: fixed;
   top: 45%;
-  left: 45%;
+  left: 49%;

```

Comment:
Just checking that this change was intentional? What is it fixing?

Reply:
The ms style buttons had more padding and margins.  So when reverting to the keyman style for buttons  ---  `Add language` and `Close`  moved towards the centre of the pop-up this adjustment counters that.